ocaml/xapi/quicktest_cbt.ml

```diff
@@ -0,0 +1,177 @@
+(*
+ * Copyright (C) 2017 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Client
+open Quicktest_common
+let start session_id =
+  let open Client in
+
+  (* Helper for test failure due to unexpected error *)
+  let report_failure error test_name test =
+    failed test (Printf.sprintf "%s failed: %s" test_name
+                   (ExnHelper.string_of_exn error)) in
+
+  let test_assert ~test op ~msg =
+    try assert op with assert_failure -> failed test msg in
+
+  (* Overall test suite runs smaller unit tests *)
+  let test_compare ~test l_op r_op ~msg =
+    let op = (l_op = r_op) in test_assert ~test op ~msg in
+
+  let name_description = "VDI for CBT quicktest" in
+  let make_vdi_from sR = (* SR has VDI.create as allowed *)
+    VDI.create ~session_id ~rpc:!rpc ~name_label:"qt-cbt" ~name_description ~_type:`user
```

Comment:
I would suggest to format this one parameter per line:
```
VDI.create ~session_id 
  ~rpc:!rpc
  ~name_description
  ~_type:`user
 ...
```

This is easier to read and easier to change. However, this is a matter of taste and just a suggestion.